htdocs/core/modules/import/modules_import.php

```diff
@@ -75,15 +75,54 @@ class ModeleImports
 
 	public $libversion = array();
 
+	/**
+	 * @var	array	Element mapping from table name
+	 */
+	public static $mapTableToElement = array(
+		'actioncomm' => 'agenda',
+		'adherent' => 'member',
+		'adherent_type' => 'member_type',
+		'bank_account' => 'bank_account',
+		'categorie' => 'category',
+		'commande' => 'commande',
+		'commande_fournisseur' => 'commande_fournisseur',
```

Comment:
Do we need record with key that has same value than value in this table ? My understanding is that this array is used to be able to get element when it is not same than table_element ? So it is a table to manage exceptions in bad old naming exception. if i'm right, can't we remove entries with key = value ?

Reply:
@eldy
I think you're right and I removed all keys are the same as table element.
And I modified method "getElementFromTableWithPrefix" to retrun "table element" as default and the mapped element if set.